User Lockout docs (#474)
clerk/clerk-docs: docs/security/unlock-user-accounts.mdx

---
title: Unlock user accounts from the Clerk Dashboard
description: Use the Clerk Dashboard to unlock user accounts.
---

# Unlock user accounts from the Clerk Dashboard

<Callout type="info">
This feature is applicable to user accounts that use [passwords](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username) or [backup codes](https://dashboard.clerk.com/last-active?path=user-authentication/multi-factor).
</Callout>

> Users with admin roles can override lockouts through the Clerk dashboard.

Reviewer: ```suggestion
Users with admin roles can override lockouts through the Clerk Dashboard.
```

> 1. In your [Clerk Dashboard](https://dashboard.clerk.com/), navigate to **[Users](https://dashboard.clerk.com/last-active?path=users)**.

Reviewer: ```suggestion
1. In your Clerk Dashboard, navigate to **[Users](https://dashboard.clerk.com/last-active?path=users)**.
```

2. Locate the locked user by looking for a "locked" badge next to their username/email. 

> 3. Open the admin menu (three little dots).

Reviewer: ```suggestion
3. Open the admin menu by selecting the icon that looks like three stacked dots.
```

> 4. Select "Unlock" to unlock the user account.

Reviewer: ```suggestion
4. Select **Unlock** to unlock the user account.
```

<Images
width={1512}
height={914}
src="/docs/images/security/userlock_dashboard.png"
alt=" "
/>

> Alternatively, you can unlock a user's account on their profile page by scrolling to the "Unlock user" section and selecting "Unlock user."

Reviewer: This is missing the step where the application owner must follow all of the above steps, and then click on "View profile", and then can scroll to the "Unlock user" section where they can select **Unlock user**.

> Alternatively, you can unlock a user's account on their profile page. Click on the user's row to be taken to their profile page, then scroll to the "Unlock user" section, and select "Unlock user."

Reviewer: ```suggestion
Alternatively, you can unlock a user's account on their profile page. 

1. In your Clerk Dashboard, navigate to **[Users](https://dashboard.clerk.com/last-active?path=users)**.
2. Locate the locked user by looking for a "locked" badge next to their username/email. 
3. Click on the user's row to be taken to their profile page.
4. Scroll to the **Unlock user** section, and select **Unlock user".
```
